ajax/dashboard.php

```diff
@@ -209,7 +209,8 @@
             } catch (\Throwable $e) {
                 // Send exception to logger without actually exiting.
                 // Use quiet mode to not break JSON result.
-                ErrorHandler::getInstance()->handleException($e, true);
+                ErrorUtils::logException($e);
+                ErrorUtils::outputExceptionMessage($e);
```

Comment:
It seems there are multiple places where a single line is replaced with 2 new ones. That's not really easy to understand and may be confusing.

Also, many things have changed regarding errro management, maybe a £ explaining how we should handle them correctly in the developper documentation would be a good thing.

Reply:
This double call is going to change. PR is still a draft.

I agree that the documentation should be update, I will take care of this once this PR will be finished.